### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by yavetisyan**

### CODE REVIEW

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/Report.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import { Link } from "react-router-dom";
+
+const Report = () => (
+  <Box className="report__box">
+    <Link to={""}>See Report</Link>
+    <Link to={""}>Restart</Link>
+  </Box>
+);
+
+export default Report;
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/Report.tsx:12]: fix

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+    stabilizedThis.sort((a, b) => {
+      const order = comparator(a[0], b[0])
+      if (order !== 0) {
+        return order
+      }
+      return a[1] - b[1]
+    })
+    return stabilizedThis.map((el) => el[0])
+  }
+
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/TableContent.tsx:155]: This same function is in 3 PR s

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    name: 'evaluators',
+    label: 'Evaluators',
+  },
+  {
+    id: 'evaluates',
+    name: 'evaluates',
+    label: 'Evaluates',
+  },
+  {
+    id: uuidv4(),
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/TableHeader.tsx:41]: tbhis too

File: src/pages/events/eventsInfo/ongoingEvent/eventTable/TableHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+]
+
+interface Asd {
+  valueToOrderBy: string | number
+  orderDirection: Order
+  handleRequestSort: (event: React.MouseEvent, property: keyof RowsInfo) => void
+}
+
+const TableHeader = ({valueToOrderBy, orderDirection, handleRequestSort}: Asd) => {
+  const classes = useEventsStyle()
```
Comment (ArthurO2002) [inline src/pages/events/eventsInfo/ongoingEvent/eventTable/TableHeader.tsx:54]: fix interfavce name

**Review by ArthurO2002 — CHANGES_REQUESTED** [review]

File: src/assets/images/auth/ChangePassword.tsx [hunk 5]
```diff
@@ -119,4 +119,4 @@ const ChangePassword = () => (
   </svg>
 )
 
-export default ChangePassword
+export default ChangePassword;
```
[the remark this answers is not in the document]
Comment (yavetisyan) [reply src/assets/images/auth/ChangePassword.tsx:122]: Done

File: src/pages/sidebar/Sidebar.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ const Sidebar = () => {
             <AvatarIcon />
           </Avatar>
           <Typography className={classes.nameSurname}>Name Surname</Typography>
-          <NavLink style={{textDecoration: 'none'}} to="/">
+          <NavLink className="text__decoration_none" to={'/'}>
             <Button variant="contained" size="large" className={globalClasses.button}>
```
Comment (ArthurO2002) [inline src/pages/sidebar/Sidebar.tsx:44]: text-decoration-none

Comment (yavetisyan) [reply src/pages/sidebar/Sidebar.tsx:44]: Done

File: src/assets/style/events/events.scss [hunk 7]
```diff
@@ -68,7 +68,7 @@
     }
   }
 
-  &__date {
+  &-date {
```
Comment (ArthurO2002) [inline src/assets/style/events/events.scss:71]: this too

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**